scripts/import.sh

```diff
@@ -1,7 +1,26 @@
-bin/rake db:drop db:create
-sudo -u postgres createuser ktom
+#!bin/bash
+unameOut="$(uname -s)"
+case "${unameOut}" in
+  Linux*)     machine=Linux;;
+  Darwin*)    machine=Mac;;
+  CYGWIN*)    machine=Cygwin;;
+  MINGW*)     machine=MinGw;;
+  *)          machine="UNKNOWN:${unameOut}"
+esac
+if [ "Mac" = "${machine}" ]; then
+  bundle exec rake db:drop db:create
+  createuser ktom
 
-cd import
-cat * | sudo -u postgres psql --set ON_ERROR_STOP=on -1 kontes_terbuka
+  cd import/db_backup/210616_000001
```

Comment:
This shouldn't be here I think

Reply:
We need this to specify from where to import right?
Or do we make `/db_backup/210616_000001` as user input instead?

Reply:
I think the idea is to copy the contents of 210616_000001 directly into the db_backup folder. So no nesting.